pollination-apps/app-developers/integrating-the-pollination-viewer.md

```diff
@@ -0,0 +1,131 @@
+## Pollination Viewer Usage
+
+Pollination App developers can easily integrate the same high performance 3D web viewer that we use throughout our apps in their own work. We offer both a Streamlit component and a React component that can be used by the Pollination community.
+
+The Streamlit component is [published on PyPi](https://pypi.org/project/pollination-streamlit-viewer/), and our React component is [available on npmjs](https://www.npmjs.com/package/lavender-vtkjs).
+
+The Streamlit component can be installed like this:
+```shell
+$ pip install pollination-streamlit-viewer
+```
+
+This guide will focus on the Streamlit component because that what will be most useful to developers of Pollination Apps. If you're interested in learning more about the React component, please reach out through our forums.
+
+The first resource you should know about is the sample viewer app: </br>
+https://app.pollination.cloud/pollination/apps/viewer-sample-app/?tab=live+app
+
+The code for this app is located here: </br>
+https://github.com/pollination-apps/pollination-viewer-sample/blob/main/app.py
```

Comment:
This is great that you are providing the link. You probably want to remove these two and add them to `.gitignore`:

![image](https://user-images.githubusercontent.com/2915573/171248513-83f39f11-6102-4623-96a8-3e13a6fd4978.png)